contrib/src/main/java/com/datatorrent/contrib/enrich/AbstractEnricher.java

```diff
@@ -73,10 +73,11 @@
   /**
    * Helper variables.
    */
-  private transient CacheManager cacheManager;
   protected transient List<FieldInfo> lookupFieldInfo = new ArrayList<>();
   protected transient List<FieldInfo> includeFieldInfo = new ArrayList<>();
 
+  private CacheManager cacheManager = new NullValuesCacheManager();
```

Comment:
Enricher is initializing cacheManager during activate but doesn't seem to be closing it in deactivate.